WordPress/src/main/java/org/wordpress/android/ui/stories/StoryComposerActivity.kt

```diff
@@ -221,4 +235,21 @@ class StoryComposerActivity : ComposeLoopFrameActivity(), SnackbarProvider, Medi
         addingMediaToEditorProgressDialog = progressDialogHelper
                 .updateProgressDialogState(this, addingMediaToEditorProgressDialog, uiState, uiHelpers)
     }
+
+    override fun getAuthHeaders(url: String): Map<String, String> {
+        return authenticationUtils.getAuthHeaders(url)
+    }
+
+    // NotificationIntentLoader
+    override fun loadIntentForErrorNotification(): Intent {
```

Comment:
Another seemingly unrelated change - this is so the stories library can properly load the intent for an error notification in case it needs to show one. For example, if you don't have the right auth headers for loading media from a private site - alternatively try adding remote media to the story, and then turning airplane mode on (haven't tried but I think it should work the same way). Then once the error notification is shown, tapping on it will trigger the action defined by such intent loaded here. Feel free to leave this part of the review to @jd-alexander @malinajirka  🙇

Reply:
I tried to test this, but I failed. The error notifications never shows up for me.